src/app/modules/admin-shared/dynamic-form/dynamic-form.component.html

```diff
@@ -6,11 +6,9 @@
         </div>
```

Comment:
Please create a component folder and put all shared components inside that

Reply:
changed